move_together_app/lib/core/models/trip.dart

```diff
@@ -20,7 +21,13 @@ class Trip {
     required this.endDate,
     required this.participants,
     required this.inviteCode,
-  });
+    this.photoUrl = '',
+    });
+
+  @override
+  String toString() {
+    return 'Trip{id: $id, name: $name, country: $country, city: $city, startDate: $startDate, endDate: $endDate, participants: ${participants.toString()}, inviteCode: $inviteCode, photoUrl: $photoUrl}';
+  }
```

Comment:
```suggestion
    required this.photoUrl,
    });
```

Du coup tu peux mettre la prop en `required`

Reply:
au final j'enleve la valeur "photoUrl" du model vu qu'elle n'est plus renvoyé coté backend